managed/services/qan/client.go

```diff
@@ -541,10 +541,20 @@ func fillPostgreSQL(mb *qanpb.MetricsBucket, bp *agentpb.MetricsBucket_PostgreSQ
 	mb.MTempBlksWrittenCnt = bp.MTempBlksWrittenCnt
 	mb.MTempBlksWrittenSum = bp.MTempBlksWrittenSum
 
-	mb.MBlkReadTimeCnt = bp.MBlkReadTimeCnt
-	mb.MBlkReadTimeSum = bp.MBlkReadTimeSum
-	mb.MBlkWriteTimeCnt = bp.MBlkWriteTimeCnt
-	mb.MBlkWriteTimeSum = bp.MBlkWriteTimeSum
+	mb.MSharedBlkReadTimeCnt = bp.MSharedBlkReadTimeCnt
+	mb.MSharedBlkReadTimeSum = bp.MSharedBlkReadTimeSum
+	mb.MSharedBlkWriteTimeCnt = bp.MSharedBlkWriteTimeCnt
+	mb.MSharedBlkWriteTimeSum = bp.MSharedBlkWriteTimeSum
+	mb.MLocalBlkReadTimeCnt = bp.MLocalBlkReadTimeCnt
+	mb.MLocalBlkReadTimeSum = bp.MLocalBlkReadTimeSum
+	mb.MLocalBlkWriteTimeCnt = bp.MLocalBlkWriteTimeCnt
+	mb.MLocalBlkWriteTimeSum = bp.MLocalBlkWriteTimeSum
+
+	// Fill deprecated fields for compatibility
+	mb.MBlkReadTimeCnt = bp.MSharedBlkReadTimeCnt
+	mb.MBlkReadTimeSum = bp.MSharedBlkReadTimeSum
+	mb.MBlkWriteTimeCnt = bp.MSharedBlkWriteTimeCnt
+	mb.MBlkWriteTimeSum = bp.MSharedBlkWriteTimeSum
```

Comment:
I think it should work the opposite way, because we are writing to DB

Reply:
Discussed on call. Filling these fields are redundant, cause we keep QAN in sync with other pmm components, so it's not possible to use old QAN API and new managed.